src/lib.rs

```diff
@@ -2467,6 +2450,54 @@ impl Build {
             println!("{}", s);
         }
     }
+
+    #[cfg(target_os = "macos")]
```

Comment:
Could this be done with `if cfg!(...)` rather than `#[cfg]`?

Reply:
Done